src/data.js

```diff
@@ -18346,6 +18346,34 @@ module.exports = [
       'REST',
     ],
   },
+  {
+    name: 'Eternal',
+    description:
+      "I’m a self-taught developer with a passion for open-source software. Additionally, I’m a hardware engineer, experimental blogger, and beginner Linux system administrator.",
+    url: 'https://3tnl.dev/uses/',
```

Comment:
Looks like this domain is down. Will you double-check? 

Sorry about the delay